### PR TITLE
Frontend/link to concordia 375

### PR DESCRIPTION
**Pull request by Vatika1**
This branch links Concordia logo on the header to the Concordia homepage website: www.concordia.ca

### CODE REVIEW

File: Client/src/Components/NavigationMenu.tsx [hunk 1]
```diff
@@ -55,7 +55,7 @@ export default function NavigationMenu() {
     return user && user.firstName ?
       (
         <Typography>
-          Hello, {user.firstName} {user.lastName}
+          {user.firstName} {user.lastName}&nbsp;&nbsp;&nbsp;
```
Comment (ds161833) [inline Client/src/Components/NavigationMenu.tsx]: remove the special symbols, do it with the help of a grid instead, check how grid is done in DefaultFormFooter for example

**Review by ds161833 — CHANGES_REQUESTED** [review]
left a comment

**Review by GD-Dheer — APPROVED** [review]
LGTM! thx

File: Client/src/Components/NavigationMenu.tsx (hunk 1 above)
Comment (Vatika1) [reply Client/src/Components/NavigationMenu.tsx]: @ds161833  done

**Review by laithAwad — APPROVED** [review]

**Review by ds161833 — APPROVED** [review]

**Pull request merged**